templates/encryption-config.yml.j2

```diff
@@ -0,0 +1,14 @@
+{% if cluster.encryptionConfig is defined and 'enabled' in cluster.encryptionConfig and cluster.encryptionConfig.enabled %}
+apiVersion: apiserver.config.k8s.io/v1
+kind: EncryptionConfiguration
+resources:
+  - resources:
+    - secrets
```

Comment:
Requires two more spaces of indentation

Reply:
Could you please indicate which line should be indentation? :)
From my point of view, they are all ok.

Reply:
https://github.com/Flaconi/devops-docs/blob/master/doc/conventions/style-guide-yaml.md#indentation

Reply:
Updated